Authenticating with public key "Imported-Openssh-Key: C:\Users\Ravi Shankar\Desktop\Docker1.pem"
     +--------------------------------------------------------------------+
     ¦                        • MobaXterm 11.1 •                          ¦
     ¦            (SSH client, X-server and networking tools)             ¦
     ¦                                                                    ¦
     ¦ ? SSH session to ec2-user@54.89.237.220                            ¦
     ¦   • SSH compression : ?                                            ¦
     ¦   • SSH-browser     : ?                                            ¦
     ¦   • X11-forwarding  : ?  (disabled or not supported by server)     ¦
     ¦   • DISPLAY         : 192.168.43.126:0.0                           ¦
     ¦                                                                    ¦
     ¦ ? For more info, ctrl+click on help or visit our website           ¦
     +--------------------------------------------------------------------+

Last login: Fri Mar 22 09:43:20 2019 from 27.59.22.221
[ec2-user@ip-172-31-89-198 ~]$ sudo -i
[root@ip-172-31-89-198 ~]# ls -lrth
total 20K
-rw-------. 1 root root 6.6K Oct 17 12:27 original-ks.cfg
-rw-------. 1 root root 7.2K Oct 17 12:27 anaconda-ks.cfg
-rw-r--r--. 1 root root   37 Mar 22 09:36 inventory
[root@ip-172-31-89-198 ~]# touch file.yaml
[root@ip-172-31-89-198 ~]# vi file.yaml
[root@ip-172-31-89-198 ~]# ansible-playbook -i inventory file.yaml
ERROR! 'state' is not a valid attribute for a Task

The error appears to have been in '/root/file.yaml': line 4, column 5, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

  tasks:
  - name: Create a new file
    ^ here

This error can be suppressed as a warning using the "invalid_task_attribute_failed" configuration
[root@ip-172-31-89-198 ~]# sudo -i
[root@ip-172-31-89-198 ~]# ls -lrth
total 24K
-rw-------. 1 root root 6.6K Oct 17 12:27 original-ks.cfg
-rw-------. 1 root root 7.2K Oct 17 12:27 anaconda-ks.cfg
-rw-r--r--. 1 root root   37 Mar 22 09:36 inventory
-rw-r--r--. 1 root root  106 Mar 22 11:05 file.yaml
[root@ip-172-31-89-198 ~]# vi file.yaml
[root@ip-172-31-89-198 ~]# ansible-playbook -i inventory file.yaml
ERROR! Syntax Error while loading YAML.
  did not find expected '-' indicator

The error appears to have been in '/root/file.yaml': line 4, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

   tasks:
  -  name: Create a new file
  ^ here

[root@ip-172-31-89-198 ~]# vi file.yaml
[root@ip-172-31-89-198 ~]# ansible-playbook -i inventory file.yaml
ERROR! Syntax Error while loading YAML.
  did not find expected '-' indicator

The error appears to have been in '/root/file.yaml': line 4, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

   tasks:
  -  name:  Create a new file
  ^ here

[root@ip-172-31-89-198 ~]# vi file.yaml
[root@ip-172-31-89-198 ~]# ansible-playbook -i inventory file.yaml
ERROR! Syntax Error while loading YAML.
  did not find expected '-' indicator

The error appears to have been in '/root/file.yaml': line 4, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

    tasks:
  -   name:  Create a new file
  ^ here

[root@ip-172-31-89-198 ~]# ls -lrth
total 24K
-rw-------. 1 root root 6.6K Oct 17 12:27 original-ks.cfg
-rw-------. 1 root root 7.2K Oct 17 12:27 anaconda-ks.cfg
-rw-r--r--. 1 root root   37 Mar 22 09:36 inventory
-rw-r--r--. 1 root root  118 Mar 22 11:10 file.yaml
[root@ip-172-31-89-198 ~]# vi inventory
[root@ip-172-31-89-198 ~]# ansible-playbook -i inventory file.yaml
ERROR! Syntax Error while loading YAML.
  did not find expected '-' indicator

The error appears to have been in '/root/file.yaml': line 4, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

    tasks:
  -   name:  Create a new file
  ^ here

[root@ip-172-31-89-198 ~]# vi file2.yaml
[root@ip-172-31-89-198 ~]# pwd
/root
[root@ip-172-31-89-198 ~]# vi inventory
[root@ip-172-31-89-198 ~]# vi file2.yaml
[root@ip-172-31-89-198 ~]# ansible-playbook -i inventory file.yaml
ERROR! Syntax Error while loading YAML.
  did not find expected '-' indicator

The error appears to have been in '/root/file.yaml': line 4, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

    tasks:
  -   name:  Create a new file
  ^ here

[root@ip-172-31-89-198 ~]# vi file2.yaml
[root@ip-172-31-89-198 ~]# ansible-playbook -i inventory file.yaml
ERROR! Syntax Error while loading YAML.
  did not find expected '-' indicator

The error appears to have been in '/root/file.yaml': line 4, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

    tasks:
  -   name:  Create a new file
  ^ here

[root@ip-172-31-89-198 ~]# vi file2.yaml
[root@ip-172-31-89-198 ~]# ansible-playbook -i inventory file.yaml
ERROR! Syntax Error while loading YAML.
  did not find expected '-' indicator

The error appears to have been in '/root/file.yaml': line 4, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

    tasks:
  -   name:  Create a new file
  ^ here

[root@ip-172-31-89-198 ~]# ansible-playbook -i inventory file2.yaml
ERROR! Syntax Error while loading YAML.
  mapping values are not allowed in this context

The error appears to have been in '/root/file2.yaml': line 4, column 15, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

  - name: Ansible create file if it doesn't exist example
    file: path: "/root/devops_server.txt"
              ^ here

[root@ip-172-31-89-198 ~]# vi file2.yaml
[root@ip-172-31-89-198 ~]# ansible-playbook -i inventory file2.yaml

PLAY [prakash] **************************************************************************************************************************

TASK [Gathering Facts] ******************************************************************************************************************
ok: [172.31.89.198]
fatal: [172.31.82.13]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: ssh: connect to host 172.31.82.1                    3 port 22: No route to host", "unreachable": true}

TASK [Ansible create file if it doesn't exist example] **********************************************************************************
changed: [172.31.89.198]
        to retry, use: --limit @/root/file2.retry

PLAY RECAP ******************************************************************************************************************************
172.31.82.13               : ok=0    changed=0    unreachable=1    failed=0
172.31.89.198              : ok=2    changed=1    unreachable=0    failed=0

[root@ip-172-31-89-198 ~]# ls -lrth
total 32K
-rw-------. 1 root root 6.6K Oct 17 12:27 original-ks.cfg
-rw-------. 1 root root 7.2K Oct 17 12:27 anaconda-ks.cfg
-rw-r--r--. 1 root root  118 Mar 22 11:10 file.yaml
-rw-r--r--. 1 root root   38 Mar 22 11:11 inventory
-rw-r--r--. 1 root root  152 Mar 22 11:18 file2.yaml
-rw-r--r--. 1 root root    0 Mar 22 11:18 devops_server.txt
-rw-r--r--. 1 root root   13 Mar 22 11:18 file2.retry
[root@ip-172-31-89-198 ~]# cat file2.yaml
- hosts: prakash
  tasks:
  - name: Ansible create file if it doesn't exist example
    file:
      path: "/root/devops_server.txt"
      state: touch
[root@ip-172-31-89-198 ~]# vi file.yam
[root@ip-172-31-89-198 ~]# vi file.yaml
[root@ip-172-31-89-198 ~]# ansible-playbook -i inventory file.yaml

PLAY [prakash] **********************************************************************************************************************************************

TASK [Gathering Facts] **************************************************************************************************************************************
ok: [172.31.89.198]
fatal: [172.31.82.13]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: ssh: connect to host 172.31.82.13 port 22: No route to host", "unreachable": true}

TASK [Ansible create file if it doesn't exist example] ******************************************************************************************************
changed: [172.31.89.198]
        to retry, use: --limit @/root/file.retry

PLAY RECAP **************************************************************************************************************************************************
172.31.82.13               : ok=0    changed=0    unreachable=1    failed=0
172.31.89.198              : ok=2    changed=1    unreachable=0    failed=0

[root@ip-172-31-89-198 ~]# ls -lrth
total 36K
-rw-------. 1 root root 6.6K Oct 17 12:27 original-ks.cfg
-rw-------. 1 root root 7.2K Oct 17 12:27 anaconda-ks.cfg
-rw-r--r--. 1 root root   38 Mar 22 11:11 inventory
-rw-r--r--. 1 root root  152 Mar 22 11:18 file2.yaml
-rw-r--r--. 1 root root    0 Mar 22 11:18 devops_server.txt
-rw-r--r--. 1 root root   13 Mar 22 11:18 file2.retry
-rw-r--r--. 1 root root  145 Mar 22 11:21 file.yaml
-rw-r--r--. 1 root root    0 Mar 22 11:21 tirumal
-rw-r--r--. 1 root root   13 Mar 22 11:21 file.retry
[root@ip-172-31-89-198 ~]# cat file2.yaml
- hosts: prakash
  tasks:
  - name: Ansible create file if it doesn't exist example
    file:
      path: "/root/devops_server.txt"
      state: touch
[root@ip-172-31-89-198 ~]# vi file.yaml
[root@ip-172-31-89-198 ~]# vi file3.yaml
[root@ip-172-31-89-198 ~]# ansible-playbook -i inventory file3.yaml

PLAY [prakash] **********************************************************************************************************************************************

TASK [Gathering Facts] **************************************************************************************************************************************
ok: [172.31.89.198]
fatal: [172.31.82.13]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: ssh: connect to host 172.31.82.13 port 22: No route to host", "unreachable": true}

TASK [Ansible create file if it doesn't exist example] ******************************************************************************************************
ok: [172.31.89.198] => {
    "msg": "Laddu"
}
        to retry, use: --limit @/root/file3.retry

PLAY RECAP **************************************************************************************************************************************************
172.31.82.13               : ok=0    changed=0    unreachable=1    failed=0
172.31.89.198              : ok=2    changed=0    unreachable=0    failed=0

[root@ip-172-31-89-198 ~]# cat file3.yaml
---
- hosts: prakash
  vars:
    hello: Laddu
  tasks:
  - name: Ansible create file if it doesn't exist example
    debug:
      msg: "{{ hello }}"

[root@ip-172-31-89-198 ~]# cat file2.yaml
- hosts: prakash
  tasks:
  - name: Ansible create file if it doesn't exist example
    file:
      path: "/root/devops_server.txt"
      state: touch
[root@ip-172-31-89-198 ~]#
